### PR TITLE
LOOP-1489: Add insulin model to therapy settings flow

### PR DESCRIPTION
**Pull request by novalegra**
https://tidepool.atlassian.net/browse/LOOP-1489

### CODE REVIEW

File: TidepoolServiceKit/Mocks/MockPrescriptionManager.swift [hunk 1]
```diff
@@ -72,7 +72,7 @@ public class MockPrescriptionManager {
                 insulinSensitivitySchedule: insulinSensitivitySchedule,
                 carbRatioSchedule: carbRatioSchedule,
                 basalRateSchedule: basalRateSchedule,
-                insulinModel: StoredSettings.InsulinModel(modelType: .rapidAdult, actionDuration: .hours(6), peakActivity: .hours(3))
+                insulinModel: InsulinModelSettings(model: ExponentialInsulinModelPreset.humalogNovologAdult)
```
Comment (novalegra) [inline TidepoolServiceKit/Mocks/MockPrescriptionManager.swift:75]: I changed `TherapySettings` to be backed by `InsulinModelSettings` for the insulin model

File: TidepoolServiceKitUI/View Controllers/PrescriptionReviewUICoordinator.swift [hunk 2]
```diff
@@ -50,13 +52,18 @@ enum PrescriptionReviewScreen {
...
+            return .insulinModelEditor
+        case .insulinModelEditor:
             return nil
         }
     }
 }
 
 class PrescriptionReviewUICoordinator: UINavigationController, CompletionNotifying, UINavigationControllerDelegate {
     var screenStack = [PrescriptionReviewScreen]()
+    var appName = "Tidepool Loop" // TODO: don't hardcode this?
```
Comment (mpangburn) [inline TidepoolServiceKitUI/View Controllers/PrescriptionReviewUICoordinator.swift]: Should disappear once `appName` is propagated via environment, wouldn't worry about it for now

**Review by mpangburn — APPROVED** [review]
LGTM!

**Review by rickpasetto — APPROVED** [review]
LGTM 🌵

**Pull request merged**